internal/context/ausf_context_init.go

```diff
@@ -40,19 +53,35 @@ func InitAusfContext(context *AUSFContext) {
 			context.UriScheme = models.UriScheme_HTTP
 		}
 
-		context.BindingIPv4 = os.Getenv(sbi.BindingIPv4)
-		if context.BindingIPv4 != "" {
-			logger.InitLog.Info("Parsing ServerIPv4 address from ENV Variable.")
+		context.BindingIPv6 = os.Getenv(sbi.BindingIPv6)
```

Comment:
maybe simplfy as such?

```go
context.BindingIPv6 = os.Getenv(sbi.BindingIPv6)
if context.BindingIPv6 != "" {
    logger.InitLog.Info("Parsing ServerIPv6 address from ENV Variable.")
} else {
    context.BindingIPv6 = sbi.BindingIPv6
}

if context.BindingIPv6 == "" {
    context.BindingIPv4 = os.Getenv(sbi.BindingIPv4)
    if context.BindingIPv4 != "" {
        logger.InitLog.Info("Parsing ServerIPv4 address from ENV Variable.")
    } else {
        context.BindingIPv4 = sbi.BindingIPv4
    }
    
    if context.BindingIPv4 == "" {
       logger.InitLog.Warn("Error parsing ServerIPv4 address as string. Using the 0.0.0.0 address as default.")
       context.BindingIPv4 = "0.0.0.0"
    }
}
```

Reply:
I was hesitant, because I not like the repetition of `if context.BindingIPv6`. It's confusing if we do not read what does the first else condition.
It's why I suggest a assignation by function